docs/articles/tasks.md

```diff
@@ -0,0 +1,35 @@
+# Tasks
+
+### What is a Task?
+
+Task is a process which is launched by Mesos executor on the Mesos Agent. When Marathon launches an instance of the application definition, Marathon sends a command to Mesos to launch one or more tasks.
```

Comment:
> Task is a process which is launched by Mesos executor on the Mesos Agent

A new user can probably imagine waht "Mesos Agent" is but what is Mesos executor? Why is it needed? 

>  Marathon sends a command to Mesos to launch one or more tasks

I feel that this too simple for what actually happens under the hood. If a user is new to Mesos, this is not particularly helpful.